### DISCUSSION TITLE
Dexcom One+ supported

### DISCUSSION THREAD

**Discussion by TWilhoWI** [NightscoutFoundation/xDrip#3336] [Q&A]
Hi all,
Got this in my e-mail today. Does anyone know of it is supported in XDrip+? Set-up process like G7?
https://www.dexcom.com/nl-be/dexcom-one-plus

**Comment by Navid200** [comment]
If you get one, try it like this: https://navid200.github.io/xDrip/docs/Dexcom/G7.html

If it doesn't work, we will need samples.

**Reply by TWilhoWI** [reply]
So not official supported? Don't know if there was much difference between G6 and One?
